<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsChamferFilletUserInputBase</class>
 <widget class="QWidget" name="QgsChamferFilletUserInputBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QgsDoubleSpinBox" name="mValue1SpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QgsDoubleSpinBox" name="mValue2SpinBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mVal1Label">
     <property name="text">
      <string>Distance</string>
     </property>
     <property name="buddy">
      <cstring>mValue1SpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mOperationLabel">
     <property name="text">
      <string>Operation</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>mOperationComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mVal2Label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Fillet segments</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>mValue2SpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="mOperationComboBox"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="mVal1Locker">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/images/themes/default/unlocked.svg</normaloff>
       <normalon>:/images/themes/default/locked.svg</normalon>:/images/themes/default/unlocked.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="mVal2Locker">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/images/themes/default/unlocked.svg</normaloff>
       <normalon>:/images/themes/default/locked.svg</normalon>:/images/themes/default/unlocked.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
